content-metadata-protobuf/test/channel.ts

```diff
@@ -0,0 +1,27 @@
+import { ChannelMetadata } from '../src'
```

Comment:
Where do these tests come from? are they autogenerated? if not, does it make sense to write tests for autogenerate code?

Reply:
I wrote them as a way to demo how to use the messages, and also as a spec to detect breaking changes when the message format is updated.